src/api/app/components/bs_request_activity_timeline_component.rb

```diff
@@ -27,7 +26,7 @@ def diff_for_ref(comment)
     return unless (ref = comment.diff_ref&.match(/diff_([0-9]+)/))
 
     file_index = ref.captures.first
-    sourcediff = @diffs.find { |d| d[:id] == comment.commentable.id }[:sourcediff].first
+    sourcediff = @commented_actions.find { |d| d.id == comment.commentable.id }.webui_sourcediff.first
```

Comment:
```suggestion
    sourcediff = @commented_actions.find(comment.commentable).webui_sourcediff.first
```
I belive they are equivalent but it needs to be tested